scripts/baseline.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+import argparse
+from src.baseline_model import Baseline
+from src.evaluate import evaluate_model 
+
+
+def main(args: argparse.Namespace):
+    df = pd.read_csv(args.data)
+    df["yo_words"] = df["yo_words"].apply(eval)
+
+    model = Baseline()
+    evaluate_model(model=model, data=df, verbose=True)
```

Comment:
Вообще не критично, но, строго говоря, тут можно не складывать в переменную:
```suggestion
    evaluate_model(model=Baseline(), data=df, verbose=True)
```